test/images/windows-servercore-cache/VERSION

```diff
@@ -1 +1 @@
-1.0
+1.1
```

Comment:
This version should never be incremented, we're never publishing this image. It's only supposed to exist in staging, from which we pull the staging image to get some cached DLLs.

Reply:
Updated.